### PR TITLE
Convert UpdatesWithDate.js to GraphQL v2

### PR DESCRIPTION
**Pull request by SudharakaP**
Resolve https://github.com/opencollective/opencollective/issues/3918

**Screen-cast**

Just a small screen-cast to show that the fetch works as expected. 😄 

Peek 2021-06-08 11-45

### CODE REVIEW

File: components/UpdatesWithData.js [hunk 1]
```diff
@@ -85,40 +87,39 @@ class UpdatesWithData extends React.Component {
...
-      }
-      fromCollective {
-        id
-        type
-        name
-        slug
-        imageUrl
+const updatesQuery = gqlV2/* GraphQL */ `
+  query Updates($collectiveSlug: String!, $limit: Int, $offset: Int) {
+    account(slug: $collectiveSlug, throwIfMissing: false) {
```
Comment (Betree) [inline components/UpdatesWithData.js:92]: For Apollo caching

```suggestion
    account(slug: $collectiveSlug, throwIfMissing: false) {
      id
```

**Review by Betree — COMMENTED** [review]
Nice!

There still seems to be an issue with pagination in some cases (https://staging.opencollective.com/gitea/updates):

Peek 2021-06-09 13-13

File: components/UpdatesWithData.js (hunk 1 above)
Comment (SudharakaP) [reply components/UpdatesWithData.js:92]: Ops, I forgot this. Added it. 👍🏽

**Comment by SudharakaP** [comment]
> Nice!
> 
> There still seems to be an issue with pagination in some cases (https://opencollective-frontend-dygxdfogu-opencollective.vercel.app/gitea/updates):
> 
> Peek 2021-06-09 13-13

@Betree : Nice catch, but this isn't specific to this PR. I ran some tests and it seems this happens for the specific case when there's exactly 20 updates (over 20 updates is the exact limit where we show the load more button). When it's 21 or more updates it works fine. But when it's exactly 20 updates the problem is that we show the button but there's nothing more to load. I'll open a new issue and PR for this. 😄

**Review by Betree — APPROVED** [review]

**Pull request merged**